plugins/org.eclipse.fordiac.ide.product/org.eclipse.fordiac.ide.product.target

```diff
@@ -59,11 +59,13 @@
 			<unit id="org.eclipse.sirius.samples.feature.group" version="7.4.0.202403051613"/>
 			<unit id="org.eclipse.sirius.specifier.feature.group" version="7.4.0.202403051613"/>
 			<unit id="org.eclipse.sirius.specifier.properties.feature.feature.group" version="7.4.0.202403051613"/>
+
 			<unit id="org.eclipse.swtbot.eclipse.feature.group" version="4.2.0.202405291350"/>
 			<unit id="org.eclipse.swtbot.eclipse.gef.feature.group" version="4.2.0.202405291350"/>
 			<unit id="org.eclipse.swtbot.feature.group" version="4.2.0.202405291350"/>
 			<unit id="org.eclipse.swtbot.ide.feature.group" version="4.2.0.202405291350"/>
 			<unit id="org.eclipse.xtext.sdk.feature.group" version="2.35.0.v20240528-1523"/>
+			<unit id="org.eclipse.swtbot.generator.feature.feature.group" version="0.0.0"/>
```

Comment:
for the production code, we need to review whether we can add this to the repository. I'd be in favor of it because it could promote creating more tests in the future. What do you think @azoitl ?

Reply:
After quickly reading the intro of swtbot generator I agree that this is something very useful. As long as it is only in the target platform but not referenced in any product related feature or plugin it should not get included into the 4diac IDE package. But a test would be good to ensure that. 

Please also add the version to be used. While we are not using versions in our manifests (please correct the manifest below in that respect) we have the dedicated versions for nearly everything in our product target file. Currently the only exception are the GEF Classic features. The reason for that is that we are beta testers of the latest GEF Classic developments and with providing 0.0. we get always the latest. For things from simrel like swtbot generator I find that not as good.